plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -95,6 +95,35 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         }
     }
 
+    /**
+     * Test that we are correctly escaped the wildcard characters while requesting DataBaseMetaData: getSchemas and getTables with params contains pattern.
+     */
+    @Test
+    public void testDatabaseMetadataSearchEscapedWildCardCharacters()
+    {
+        // wildcard characters on schema name
+        assertQuerySucceeds("SHOW TABLES FROM public");
+        assertQueryFails("SHOW TABLES FROM \"publi_\"", ".*Schema 'publi_' does not exist");
+        assertQueryFails("SHOW TABLES FROM \"pu%lic\"", ".*Schema 'pu%lic' does not exist");
+
+        try {
+            assertUpdate("CREATE TABLE abcd (a int, b int, c int) WITH (primary_key = ARRAY['a'])");
```

Comment:
Rename "abcd" to "test_search" or something and add random prefix.

Reply:
renamed and added a random suffix.